apps/ledger-live-desktop/src/renderer/screens/account/index.jsx

```diff
@@ -93,6 +93,18 @@ const AccountPage = ({
 
   const isCompoundEnabled = useCompoundAccountEnabled(account, parentAccount);
 
+  const [banner, setBanner] = useState({});
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const bannerProps = await getBannerProps(account, { t, dispatch });
+      setBanner(bannerProps);
+    };
+    fetchData();
```

Comment:
?
```suggestion
if(account && mainAccount) {
    fetchData();
}
```

Reply:
I made `getBannerProps` not async so I think this part should be good now. :+1: